src/views/saphyr/SaphyrReception.vue

```diff
@@ -48,20 +48,31 @@ export default {
     async handleSampleExtractionTubes() {
       try {
         let getSETubeResponse = await this.getSampleExtractionTubesForBarcodes(this.getBarcodes())
-        if (!getSETubeResponse.successful || getSETubeResponse.empty) {
-          throw getSETubeResponse.errors
+
+        if (!getSETubeResponse.success || getSETubeResponse.data.data.length === 0) {
```

Comment:
Based on the associated changes in the action this could now be:

```
if (!getSETubeResponse.success) {
 throw { message: getSETubeResponse.errors }
}
```

Reply:
Thanks for these suggestions